src/style.css

```diff
@@ -54,19 +54,20 @@
 
   .DayPicker-Weekdays {
     display: table-header-group;
-    > div {
```

Comment:
I see! CSS is OKish with nested selectors by silently dropping them =(. Loved that feature for years.